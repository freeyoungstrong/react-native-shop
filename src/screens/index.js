export * from './login/login';
export * from './main/main';
export * from './productDetails/productDetails';
export * from './registration/registration';
export * from './welcome/welcome';
export * from './map/map';
