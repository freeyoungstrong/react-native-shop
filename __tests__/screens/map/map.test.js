import React from 'react';
import { shallow } from 'enzyme';

import { MapScreen } from 'screens';

describe('Map screen', () => {
    let wrapper;

    beforeEach(() => {
        wrapper = shallow(<MapScreen />);
    });

    it('should render fine', () => {
        expect(wrapper).toMatchSnapshot();
    });
});
