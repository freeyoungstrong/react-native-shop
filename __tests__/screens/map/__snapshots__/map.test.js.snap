// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Map screen should render fine 1`] = `
<Fragment>
  <Header
    title="Map"
  />
  <View
    style={
      Object {
        "alignItems": "center",
        "flex": 1,
        "justifyContent": "center",
      }
    }
  >
    <MapView
      provider="google"
      region={
        Object {
          "latitude": 37.7825259,
          "latitudeDelta": 0.09,
          "longitude": -122.345453,
          "longitudeDelta": 0.035,
        }
      }
      showsUserLocation={true}
      style={
        Object {
          "height": "100%",
          "width": "100%",
        }
      }
    >
      <MapMarker
        coordinate={
          Object {
            "latitude": 37.782529,
            "longitude": -122.34453,
          }
        }
        stopPropagation={false}
      >
        <MapCallout
          alphaHitTest={false}
          tooltip={false}
        >
          <Text>
            You are here!
          </Text>
        </MapCallout>
      </MapMarker>
    </MapView>
  </View>
</Fragment>
`;
